me.sergiotarxz.JapaChar.yml

```diff
@@ -0,0 +1,90 @@
+id: "me.sergiotarxz.JapaChar"
+runtime: "org.gnome.Platform"
+runtime-version: "46"
+sdk: "org.gnome.Sdk"
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --persist=.japachar
```

Comment:
Reading this https://git.owlcode.tech/sergiotarxz/JapaChar/src/branch/main/lib/JapaChar/DB.pm#L41

it's likely to be unnecessary. But again you should know better that I do.

Reply:
What you say is true, I just tested, removing it..